nyaa/routes.py

```diff
@@ -470,16 +471,30 @@ def upload():
         return flask.render_template('upload.html', form=form, user=flask.g.user), status_code
 
 
-@app.route('/view/<int:torrent_id>')
+@app.route('/view/<int:torrent_id>', methods=['GET', 'POST'])
```

Comment:
The POST route for the comments should be different; just something like `/view/<int:torrent_id>/submit_comment` etc - let's not hijack the main view.